### DISCUSSION TITLE
Is Tiger 2023 available?

### DISCUSSION THREAD

**Discussion by lucasmaj** [mediagis/nominatim-docker#539] [General]
I noticed Tiger file tiger2021-nominatim-preprocessed.csv.tar.gz is used. Is it truly 2021 version and isn't there 2023 version? Could be totally wrong. Please advise.

https://nominatim.org/downloads/

**Comment by mtmail** [comment]
The file on nominatim.org/downloads is the 2023 data https://github.com/osm-search/Nominatim/issues/3258#issuecomment-1828652122

**Comment by lucasmaj** [comment]
Right, but this project is using u355874-sub1@u355874-sub1.your-storagebox.de:tiger2021-nominatim-preprocessed.csv.tar.gz (https://github.com/mediagis/nominatim-docker/blob/master/4.3/init.sh) which by name makes it look like 2021.

**Reply by mtmail** [reply]
Yes it's outdated then. I think @leonardehrenfried maintains the storagebox server.

You can download the file separately and set it as `IMPORT_TIGER_ADDRESSES` in the configuration . https://github.com/mediagis/nominatim-docker/tree/master/4.3#configuration

**Reply by lucasmaj** [reply]
That's what I will do.

**Reply by leonardehrenfried** [reply]
I've uploaded tiger 2023 to the storage box and opened a PR here: https://github.com/mediagis/nominatim-docker/pull/544
